src/backend/utils/indexer.js

```diff
@@ -67,23 +67,16 @@ const search = async (textToSearch) => {
     body: { hits },
   } = await client.search({
     from: 0,
+    _source: ['null'],
     size: ELASTIC_MAX_RESULTS || 100,
     index,
     type,
     body: query,
   });
-  const results = hits.total.value;
-  const values = hits.hits.map(({ _id, _source, _score }) => {
-    return {
-      id: _id,
-      text: _source.text,
-      score: _score,
-    };
-  });
 
   return {
-    results,
-    values,
+    results: hits.total.value,
+    values: hits.hits.map(({ _id }) => ({ id: _id })),
```

Comment:
In a PR, only fix what you have to fix. Don't refactor. I would leave the code as is and just remove the `text: _source.text` and `score: _score` line.

```
const values = hits.hits.map(({ _id }) => {	
      id: _id,	
    };	
  });
```

or something like that

Reply:
That's @humphd's idea so I changed it. Do you want me to roll back?

Reply:
No keep it as is.